scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScaladocParser.scala

```diff
@@ -148,21 +152,37 @@ object ScaladocParser {
     }
   }
 
-  private def listBlockParser[_: P](minIndent: Int = 1): P[ListBlock] = P {
-    def listParser = (hspacesMinWithLen(minIndent) ~ listPrefix.! ~ hspaces1).flatMap {
+  def listItemParser[_: P](indent: Int): P[ListItem] = { 
+    def stop(indent: Int) = 
+      nl ~ hspace.rep(max = indent - 1) ~ listPrefix ~ hspaces1 ~ textParser()
+
+    def innerBody = 
+      (!stop(indent) ~ 
+        listBlockParser(indent) |
+        mdCodeBlockParser(indent) |
+        tableParser(indent) |
+        textParser(indent)
+      ).rep(0, sep = paraSep)
+    
+    (textParser() ~ innerBody)
+     .map{case (beginning, other) => ListItem(beginning +: other)}
+  }
+
+  private def listBlockParser[_: P](indent: Int = 1): P[ListBlock] = P {
```

Comment:
this `indent` conflicts with the one on line 153(left)/173(right). i used `minIndent` to avoid this clash.

Reply:
reverted it back to `minIndent`